docs/program/apis/sessions.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Session Management with Viam's Client SDKs"
+linkTitle: "Session Management API"
+weight: 20
+type: "docs"
+description: "How to use the session management API with Viam's Client SDKs."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session", "sessions", "session management"]
+---
+
+The session management API provides support for robot session management.
+
+A session:
+
+- allows a client to express that it is actively connected or authenticated to `viam-server` on your robot
+- supports stopping moving components when the connection is no longer active.
+
+### Purpose of session management
+
+When controlling a robot or fleet with Viam, you want a way to understand the presence of the clients that are communicating with and authenticated to each robot's `viam-server`.
+
+The period of time during which these clients are present is called a *session*.
+*Session management* is a presence mechanism between a client and `viam-server`.
+
+Session management allows for safer operation of robots that physically move.
+For example, imagine a wheeled rover gets a [`SetPower()`](/components/base/#setpower) command as the last input from a client before the connection to the robot is interrupted.
+Without session management, the API request from the client sets the flow of electricity to the motors of the robot and then does not time out, causing the robot to continue driving forever, colliding with objects and people.
+
+With session management, if the robot does not receive a signal from the client at regular intervals, it will safely stop until the connection is reestablished.
+
+#### Clients and Viam's client SDKs
+
+*Client* has multiple meanings for a Viam robot.
+Essentially, it's anything that is receiving the information served by `viam-server` running on the robot.
+
+A **client** can be an SDK script controlling the robot, an input controller, or just the different resources on the robot talking amongst themselves.
+For example, if you use Viam's module registry to [add modular resources to your robot](/extend/modular-resources/), the clients of your robot in its lifetime will include the "model servers" you instantiate on your robot for individual resources.
```

Comment:
```suggestion
For example, if you [add modular resources to your robot](/extend/modular-resources/), the clients of your robot include the "module servers" you instantiate on your robot for individual resources.
```
Can shorten. I'm less sure about module versus model, but it seems like module makes more sense?

Reply:
This should be kept model to be consistent with the other docs. I know it might seem like module makes more sense but the module refers to the mechanism and the model refers to the actual plans for design-- think of architecture, the docking mechanism vs the dock that is built.

Reply:
How about 
```suggestion
For example, if you use Viam's module registry to [add modular resources to your robot](/extend/modular-resources/), the clients of your robot in its lifetime will include the "modular resource servers" you instantiate on your robot for individual resources.
```
then?